docs/_scripts/advisory.sh

```diff
@@ -11,8 +11,7 @@ export ADVISORY="advisory.json"
 # Upload advisory
 echo "Upload advisory in JSON format."
 export TASK_URL=$(http --form POST $BASE_ADDR/pulp/api/v3/content/rpm/advisories/ \
-    file@./$ADVISORY repository=$REPO_HREF \
-    relative_path=$ADVISORY | jq -r '.task')
```

Comment:
Does this change remove the `relative_path` option? If so, I think this should technically have a removal changelog. Something like "Removed extraneous relative_path option when uploading an advisory."